### ISSUE TITLE
Users unable to list all assets when not having global super user rights

### ISSUE THREAD

**Issue by colinerd** [snipe/snipe-it#5103]
#### Expected Behavior (or desired behavior if a feature request)

Group "IT Users" is having all rights except for "global super user" which is set to deny. Users in the group should be able to work with the entire system and can access assets from the dashboard and asset list.

-----

#### Actual Behavior

Group "IT Users" is having all rights except for "global super user" which is set to deny. Users in the group are able to work with the entire system and can access assets from only the dashboard however when going into Assets - List all, the list remains empty "no matching records found" (there is no filter active).

One I activate the rights "global super user" for the same group, the assets becomes visible from the List asset page.

-----

#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? - Fresh Install
- Version of Snipe-IT you're running - 4.1.14
- Version of PHP you're running - 7.1.7
- Version of MySQL/MariaDB you're running - 10.1
- What OS and web server you're running Snipe-IT on - Windows Server 2016 & IIS 10
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) - Manual installation following the instructions from the website.
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error - No error available
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error - All Assets (/hardware)

- Include what you've done so far in the installation, and if you got any error messages along the way. - Played with the rights of the group but only way to have them see the full list is to give global super user rights.
- Indicate whether or not you've manually edited any data directly in the database - Not touched the database manually.

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by snipe** [comment]
Do you have full company support turned on in your settings? Anything in your /storage/logs/laravel.log?

**Comment by colinerd** [comment]
Ah yes I did, never realized during set up to be honest... I have turned it off and now it seems resolved. I did not had the user assigned to a company either which explains it.

What I do find strange is why the users were able to see and open the asset from the dashboard but not the asset list. If it is because the users were not assigned to a company, I understand the list being empty but then I would not expect it to be visible on the dashboard either? Currently the users were able to see and open the asset from the dashboard....
